pkg/docker/config/config.go

```diff
@@ -88,16 +88,16 @@ func GetAuthentication(sys *types.SystemContext, registry string) (string, strin
 // GetUserLoggedIn returns the username logged in to registry from either
```

Comment:
The comment mentions returning the username, but doesn't mention returning the password too and probably should.

Reply:
Fixed. Thanks for pointing it out :)